doc/run/k8s/install-sqlflow.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sqlflow
+
+---
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: Deployment
+metadata:
+  name: sqlflow-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sqlflow-server
+  template:
+    metadata:
+      labels:
+        app: sqlflow-server
+    spec:
+      containers:
+      - image: sqlflow/sqlflow
+        name: sqlflow-server
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 50051
+          name: sqlflow-server
+        - containerPort: 8888
+          name: notebook
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sqlflow-mysql
+spec:
+  selector:
+    matchLabels:
+      app: sqlflow-mysql
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: sqlflow-mysql
+    spec:
+      volumes: 
+      containers:
+      - image: sqlflow/sqlflow:mysql
+        name: mysql
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3306
```

Comment:
It seems that we have to change the `start.sh` in SQLFlow image to access this DB.

Reply:
Thanks, will use a specified command line instead of the default `start.sh`